documentation/examples/prometheus-kubernetes.yml

```diff
@@ -18,14 +18,14 @@ scrape_configs:
   bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
 
   kubernetes_sd_configs:
-  - masters:
+  - api_servers:
     - 'https://kubernetes.default.svc'
     in_cluster: true
 
   relabel_configs:
   - source_labels: [__meta_kubernetes_role]
     action: keep
-    regex: ^(?:master|node)$
+    regex: ^(?:apiserver|node)$
```

Comment:
By now our relabeling regexes are all anchored, i.e. wrapped with `^(?:<x>)$` internally.
I had a discussion with @grobie about sanity of this.

@grobie agreed to make a case for that in the next few days. For now let's remove the outer anchoring to match the current behavior, though questionable.